zone/gm_commands/summon.cpp

```diff
@@ -7,82 +7,79 @@ extern WorldServer worldserver;
 
 void command_summon(Client *c, const Seperator *sep)
 {
-	Mob *t;
+	int arguments = sep->argnum;
+	if (!arguments && !c->GetTarget()) {
+		c->Message(Chat::White, "Usage: #summon - Summon your target, if you have one, to your position");
+		c->Message(Chat::White, "Usage: #summon [Character Name] - Summon a character by name to your position");
+		c->Message(Chat::White, "Note: You may also summon your target if you have one.");
+		return;
+	}
+
+	Mob* target;
```

Comment:
Totally ok and actually preferred to single letter variables of complex type receivers.

Complex types meaning structs, classes.

Example, `Client *c` vs `Client *client` is not any less understandable or readable in code. It actually does make things more readable and less verbose as you access it many times in following code.

Common practice in Go: https://github.com/golang/go/wiki/CodeReviewComments#receiver-names